tools-core/src/test/java/com/hedera/hashgraph/client/core/json/IdentifierTest.java

```diff
@@ -266,7 +267,9 @@ void equals_test() {
 		assertEquals(id1, id1);
 		assertNotEquals(id1, id3);
 
+		//noinspection EqualsBetweenInconvertibleTypes
 		assertFalse(id1.equals("1.2.3"));
+		//noinspection ConstantConditions
 		assertFalse(id1.equals(null));
```

Comment:
This should have been complained by `SonarCloud`. Change it to `assertNotNull()` or assign to a boolean before asserting it to satisfy the code coverage.